cobc/ChangeLog

```diff
@@ -1,4 +1,11 @@
 
+2023-07-02  Fabrice Le Fessant <fabrice.le_fessant@ocamlpro.com>
+
+	* flag.def/cobc.c: new flags -fno-ttimestamp to suppress timestamp
+	in listing headers, and -fttitle=<TITLE>, so that listings can be
```

Comment:
please 2-space indent after the asterisk as in the other entries

Reply:
done